parachain/pallets/ethereum-beacon-client/src/mock.rs

```diff
@@ -97,6 +97,8 @@ pub mod mock_minimal {
 				epoch: 0,
 			},
 		};
+		pub const MaxSyncCommitteeCachedSize: u32 = 256;
+		pub const MaxExecutionHeaderCachedSize: u32 = 8192;
```

Comment:
These values seem too large for mock configuration. I would expect really small numbers so that it is easy to test cases where the bounds are exceeded.